lang/java/avro/src/test/java/org/apache/avro/TestSchemas.java

```diff
@@ -46,8 +46,8 @@
 
   static final Schema ENUM1_AB_SCHEMA = Schema.createEnum("Enum1", null, null, list("A", "B"));
   static final Schema ENUM1_AB_SCHEMA_DEFAULT = Schema.createEnum("Enum1", null, null, list("A", "B"), "A");
-  static final Schema ENUM1_AB_SCHEMA_NAMESPACE_1 = Schema.createEnum("Enum1", null, "namespace1", list("A", "B"));
-  static final Schema ENUM1_AB_SCHEMA_NAMESPACE_2 = Schema.createEnum("Enum1", null, "namespace2", list("A", "B"));
+  public static final Schema ENUM1_AB_SCHEMA_NAMESPACE_1 = Schema.createEnum("Enum1", null, "namespace1", list("A", "B"));
+  public static final Schema ENUM1_AB_SCHEMA_NAMESPACE_2 = Schema.createEnum("Enum1", null, "namespace2", list("A", "B"));
```

Comment:
Making these public is not a big deal; they're general test values in test code.

As a matter of personal preference, I would like to have them all public (or none, but that's not an option with the very valid use below).

What is the opinion of others here?